react/ecmascript/01 - spread/index.js

```diff
@@ -1,7 +1,54 @@
 import isArray from './utils';
 
-export function min(numbers) {
-  return Math.min(...numbers);
+export function min(...numbers) {
+  if (numbers.length !== 0) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+    if (numbers.length === 1) {
+      return Math.min(numbers);
+    }
+    const array = [];
```

Comment:
`array` ends up having the same elements as `numbers` so you can just use `numbers` instead.